src/components/Nav/modal/ProfileContainer.js

```diff
@@ -1,12 +1,25 @@
 import React from 'react';
 import styled from 'styled-components';
 
-const ProfileContainer = ({ profileModal, setProfileModal }) => {
+const ProfileContainer = ({
+  profileModal,
+  setProfileModal,
+  modalIsOpen,
+  setModalIsOpen,
+}) => {
+  const swtichModal = () => {
+    setProfileModal(prev => !prev);
+    setModalIsOpen(prev => !prev);
+  };
   return (
     <ModalOverlay onClick={() => setProfileModal(false)}>
       <ModalProfile onClick={e => e.stopPropagation()}>
         <TopContainer>
-          <UserInfoMenu>
+          <UserInfoMenu
+            onClick={() => {
+              swtichModal();
+            }}
```

Comment:
```suggestion
            onClick={swtichModal}
```
이 부분은 이렇게 수정해도 되겠습니다!

Reply:
수정했습니다:)!